tests/system_tests/test_blueapi_system.py

```diff
@@ -212,13 +286,13 @@ def test_get_task_by_status(client: BlueapiClient):
     client.clear_task(task_id=task_2.task_id)
 
 
-def test_progress_with_stomp(client_with_stomp: BlueapiClient):
+def test_progress_with_stomp(client_with_authenticated_stomp: BlueapiClient):
```

Comment:
As above make `client_with_stomp` use the authentication and have a `unauthenticated_client_with_stomp` that will fail later